browser/components/smartwindow/content/chat.mjs

```diff
@@ -436,8 +465,14 @@ class ChatBot extends MozLitElement {
       return;
     }
 
-    // Add the user message
-    this.messages.push({ role: "User", content: this.prompt });
+    // Add the user message (keep rich HTML for UI)
+    this.messages.push({
+      role: "User",
+      content: this.prompt, // plain text goes to history/model
+      displayHTML: this._lastUserHTML || null, // rich chip for UI only
```

Comment:
sorry for the merge conflict with #20. @s9tpepper thoughts on how we should save @ mentions in chat history? i don't think we should be saving raw html for actual implementation but for demo this could be fine for now